src/microservices/Microservices.CohortPackager/Execution/ExtractJobStorage/MongoDB/MongoExtractJobStore.cs

```diff
@@ -44,6 +58,66 @@ public MongoExtractJobStore(IMongoClient client, string extractionDatabaseName,
 
             long count = _inProgressJobCollection.CountDocuments(FilterDefinition<MongoExtractJobDoc>.Empty);
             Logger.Info(count > 0 ? $"Connected to job store with {count} existing jobs" : "Empty job store created successfully");
+
+            if (verificationMessageQueueFlushTime.TotalMilliseconds >= int.MaxValue)
+                verificationMessageQueueFlushTime = TimeSpan.FromMilliseconds(int.MaxValue);
```

Comment:
I'm probably overlooking something, but why is this capping `TimeSpan.TotalMilliseconds` to `int.MaxValue` specifically?

Reply:
Ah, I hadn't realised this was because of the `Timer()` that follows this - it looked unrelated because of the blank line. Thanks! Yes, I think a short comment would be helpful.